presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcTableHandle.java

```diff
@@ -72,6 +82,12 @@ public String getTableName()
         return tableName;
     }
 
+    @JsonProperty
+    public TupleDomain<ColumnHandle> getTupleDomain()
```

Comment:
Maybe call this `getConstraint`? It *is* a `TupleDomain`, but its purpose is to represent the constraint associated with this (derived) table.